packages/pds-ember/app/styles/pds/tokens/color/index.scss

```diff
@@ -13,12 +12,11 @@
 @forward "_Vault" as Vault-*;
 @forward "_HashiCorp" as HashiCorp-*;
 
-/* ======================================== *\
- * Color Palette
-\* ======================================== */
-/*
-  As defined in the "Structure 1.x" Figma library.
-*/
+// ======================================== //
+// Color Palette
+// ======================================== //
+// As defined in the "Structure 1.x" Figma library.
+
```

Comment:
Converted to Sass comments to prevent detached CSS comments from generating in output.